WebApi/WebApiTest/MapperTest/PostMappersTest/PostMapperTest.cs

```diff
@@ -0,0 +1,124 @@
+                    new Post { PostID = 12, UserID = 8, CategoryID = 3, Date = DateTime.UtcNow, IsPromoted = true, Localization = "War sza w a", ShopName = "U Tomka" },
+                    new Post { PostID = 2, UserID = 5, CategoryID = 5, Title= "Title32321", Date = DateTime.Now, IsPromoted = false, Localization = "Krakówarszawa", ShopName = "U Pawła" },
+                    new Post()
+                }
+
+             };
+        }
+        public static IEnumerable<object[]> PostDTODataList()
+        {
+            yield return new object[]
+            {
+                new List<PostDTO>
+                {
+                    new PostDTO { title ="Title123", author = "Janek", authorID =5, category = "RTV", content="Cześć 320", datetime=DateTime.Now, id=321, isLikedByUser=false, isPromoted=false, likesCount=15 },
+                    new PostDTO { title ="Title123", author = "Janek", authorID =5, datetime=DateTime.Now, id=321, isLikedByUser=false, isPromoted=false, likesCount=15 },
+                    new PostDTO { author = "Janek", authorID =5, category = "RTV", datetime=DateTime.UtcNow, isLikedByUser=false, isPromoted=false, likesCount=15 },
+                    new PostDTO ()
+                }
+
+             };
+
+        }
+
+        public static IEnumerable<object[]> PostDTOData()
+        {
+            yield return new object[] { new PostDTO { title = "Title123", author = "Janek", authorID = 5, category = "RTV", content = "Cześć 320", datetime = DateTime.Now, id = 321, isLikedByUser = false, isPromoted = false, likesCount = 15 } };
+            yield return new object[] { new PostDTO { title = "Title123", author = "Janek", authorID = 5, datetime = DateTime.Now, id = 321, isLikedByUser = false, isPromoted = false, likesCount = 15 }, };
+            yield return new object[] { new PostDTO { author = "Janek", authorID = 5, category = "RTV", datetime = DateTime.UtcNow, isLikedByUser = false, isPromoted = false, likesCount = 15 } };
+            yield return new object[] { new PostDTO() };
+        }
+
+        public static IEnumerable<object[]> PostPOCOData()
+        {
+            yield return new object[] { new Post { PostID = 1, UserID = 1, CategoryID = 1, Title = "Title1", Content = "Content1 ", Date = DateTime.Now, IsPromoted = false, Localization = "Warszawa", ShopName = "U Romka" } };
+            yield return new object[] { new Post { PostID = 12, UserID = 8, CategoryID = 3, Date = DateTime.UtcNow, IsPromoted = true, Localization = "War sza w a", ShopName = "U Tomka" } };
+            yield return new object[] { new Post { PostID = 2, UserID = 5, CategoryID = 5, Title = "Title32321", Date = DateTime.Now, IsPromoted = false, Localization = "Krakówarszawa", ShopName = "U Pawła" } };
+            yield return new object[] { new Post() };
+        }
```

Comment:
Nie używamy zmiennych DateTime które będą się różniły na różnych serwerach (bo tak to na prawdę będzie wyglądało). Zmienić żeby czas był na sztywno wpisany.